srv/components/system/network/mgmt/bonded/init.sls

```diff
@@ -15,7 +15,7 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-{%- if (pillar['cluster'][node]['network']['mgmt_nw']['ipaddr']) and (not pillar['cluster'][grains['id']]['network']['mgmt_nw']['gateway']) %}
+{%- if (pillar['cluster'][node]['network']['mgmt'ipaddr']) and (not pillar['cluster'][grains['id']]['network']['mgmmgmtteway']) %}
```

Comment:
This is incorrect, can you check if we need to update:
['mgmt]['public_ip']
['mgmt']['gateway']